DelvUI/Interface/GeneralElements/PrimaryResourceHud.cs

```diff
@@ -44,14 +44,30 @@ public override void DrawChildren(Vector2 origin)
             var drawList = ImGui.GetWindowDrawList();
             drawList.AddRectFilled(startPos, startPos + Config.Size, 0x88000000);
 
-            drawList.AddRectFilledMultiColor(
+            if (Config.UseJobColor)
+            {
+                var color = GlobalColors.Instance.SafeColorForJobId(chara.ClassJob.Id);
+
+                drawList.AddRectFilledMultiColor(
+                startPos,
+                startPos + new Vector2(Math.Max(1, Config.Size.X * scale), Config.Size.Y),
+                color.TopGradient,
+                color.TopGradient,
+                color.BottomGradient,
+                color.BottomGradient
```

Comment:
I missed changing these calls to use the DrawHelper.DrawGradientFilledRect earlier, can you replace them with that? So it uses the global gradient setting

Reply:
Also instead of having the same code twice, define the color first, then make the call, like you did in LabelHud

Reply:
Sure can sort it when daughter is asleep later today

Reply:
Ok sorted, also added the call to DrawHelper.DrawGradientFilledRect for the FriendlyNPC unitframe as well since I noticed it was missing there too